funnel/templates/macros.html.jinja2

```diff
@@ -109,46 +106,51 @@
-        {%- endif %}
-        {%- if project.primary_venue %}
-          <a class="mui--text-hyperlink mui--text-subhead project-venue {% if current_page == 'project' %}js-smooth-scroll{%- endif %}" href="{%- if current_page != 'project' -%}{{ project.url_for() }}{%- endif %}#venue" data-action="Venue(header)" data-action="Venue(header)">{% assets "fa5-sprite" %}<svg class="fa5-icon fa5-title fa5--align-baseline" aria-hidden="true" role="img"><use xlink:href="{{ ASSET_URL }}#map-marker-alt"></use></svg>{% endassets %} <span>{{ project.primary_venue.title }}</span>{%- if project.primary_venue.city %}, <span>{{ project.primary_venue.city }}</span>{%- endif %}</a>
-        {%- endif %}
-        {% if project.cfp_state.OPEN -%}
-          <p><a class="mui-btn mui-btn--raised mui--d-block propose-btn" href="{{ project.url_for('new_proposal') }}" data-action="Submit a session proposal(header)" data-cy="propose-a-session">{% trans %}Submit a session proposal{% endtrans %}</a></p>
-        {%- elif project.cfp_state.CLOSED or project.cfp_state.EXPIRED %}
-          <div>
-            <a class="mui-btn mui-btn--raised mui--d-block propose-btn strike-text" data-cy="Submit a session proposal" data-action="Propose a session(closed)">{% trans %}Propose a session{% endtrans %}</a>
-            <p class="mui--text-light mui--text-caption mui--text-center">Submissions are closed for this project</p>
-          </div>
-        {%- endif %}
-        <div class="mui--hidden-xs mui--hidden-sm">
-          {%- if project.allow_rsvp or project.boxoffice_data and project.boxoffice_data.item_collection_id -%}
-            <a href="{%- if current_page == 'project' -%}#tickets{%- else %}{{ project.url_for() }}#tickets{%- endif %}" class="mui-btn mui-btn--raised mui-btn--primary mui--d-block {% if current_page == 'project' %}js-smooth-scroll js-samepage{%- endif %}" data-action="Get tickets(header)" id="ticket-btn">{%- if project.allow_rsvp %}{% trans %}Attend{% endtrans %}{% else %}{% trans %} Get tickets{% endtrans %}{% endif %}</a>
-          {%- elif project.buy_tickets_url.url %}
-            <a class="mui-btn mui-btn--raised mui-btn--primary mui--d-block" href="{{ project.buy_tickets_url }}" data-action="Buy Tickets(header)">
-            {% if project.boxoffice_data and project.boxoffice_data.buy_btn_text -%}
-              {{ project.boxoffice_data.buy_btn_text }}
-            {% else %}
-              {% trans %}Get tickets{% endtrans %}
-            {% endif %}
-            </a>
+        <div class="project-details__box--ticketbox">
+          {%- if project.start_at %}
+            <p class="mui--text-light mui--text-subhead project-time">{% assets "fa5-sprite" %}<svg class="fa5-icon fa5-title fa5--align-baseline" aria-hidden="true" role="img"><use xlink:href="{{ ASSET_URL }}#clock"></use></svg>{% endassets %} <span>{{ project.start_at }}</span>{%- if project.end_at %} <span>–{{ project.end_at }}</span>{%- endif %}</p>
```

Comment:
Isn't this `schedule_start_at`?

Reply:
Removed it. That was the old design now we are showing the time in the calendar widget on hover.